integration/ramdisk/ramdisk.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../lib/common.bash"
+
+set -e
+
+# Environment variables
+IMAGE="${IMAGE:-busybox}"
+CONTAINER_NAME="${CONTAINER_NAME:-test}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-sleep 30}"
+
+# Set the runtime if not set already
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+# File to save ramdisk results
+TMP_FILE=$(mktemp ramdisk.XXXXXXXXXX)
+
+remove_tmp_file() {
+	rm -rf $TMP_FILE
+}
+
+trap remove_tmp_file EXIT
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	check_processes
```

Comment:
If we dictate that all tests must do their own checks at the end of them, then you might not need a pre-check here. Not a big problem, but at this rate we will end up calling the check before and after each test, which is probably theoretically twice as often as we actually need to.